src/Query/Builder.php

```diff
@@ -854,7 +854,7 @@ public function pluck($column, $key = null)
         // Convert ObjectID's to strings
         if (((string) $key) === '_id') {
             $results = $results->map(function ($item) {
-                $item['_id'] = (string) $item['_id'];
+                $item->_id = (string) $item->id;
```

Comment:
I think this piece of code can be removed as `Arr::pluck` already convert stringable key value into string. https://github.com/laravel/framework/blob/646520ad682d98b5211c6e26092259cfbe130b5c/src/Illuminate/Collections/Arr.php#L562